cmd/neofs-node/netmap.go

```diff
@@ -22,6 +22,11 @@ import (
 	"go.uber.org/zap"
 )
 
+// defaultEpochDuration sets default epoch duration. Should not be used in a
+// normal situations but if no info can be fetched, smth should be used as a
+// fallback value.
+const defaultEpochDuration = 240
```

Comment:
IIRC we have this const in IR somewhere as well.

Reply:
tried to find (looked for `240`) but no, just magic numbers in neofs-adm, IR's deploy procedure, etc. also, i am not sure where it should be placed and if we should have a single const for every application